examples/quickstart/10-configmaps.json

```diff
@@ -0,0 +1,35 @@
+{
+   "apiVersion": "v1",
+   "items": [
+      {
+         "apiVersion": "v1",
+         "data": {
+            "config.json": "{\"Description\": \"EXAMPLE\", \"Filters\": {\"LabelSelector\": \"\", \"Namespaces\": \".*\"}, \"PluginNamespace\": \"heptio-sonobuoy\", \"Plugins\": [{\"name\": \"systemd_logs\"}, {\"name\": \"e2e\"}], \"Resources\": [\"CertificateSigningRequests\", \"ClusterRoleBindings\", \"ClusterRoles\", \"ComponentStatuses\", \"Nodes\", \"PersistentVolumes\", \"PodSecurityPolicies\", \"ServerVersion\", \"StorageClasses\", \"ThirdPartyResources\", \"ConfigMaps\", \"DaemonSets\", \"Deployments\", \"Endpoints\", \"Events\", \"HorizontalPodAutoscalers\", \"Ingresses\", \"Jobs\", \"LimitRanges\", \"PersistentVolumeClaims\", \"Pods\", \"PodLogs\", \"PodDisruptionBudgets\", \"PodPresets\", \"PodTemplates\", \"ReplicaSets\", \"ReplicationControllers\", \"ResourceQuotas\", \"RoleBindings\", \"Roles\", \"ServiceAccounts\", \"Services\", \"StatefulSets\"], \"ResultsDir\": \"/tmp/sonobuoy\", \"Server\": {\"advertiseaddress\": \"sonobuoy-master:8080\", \"bindaddress\": \"0.0.0.0\", \"bindport\": 8080, \"timeoutseconds\": 3600}, \"Version\": \"v0.8.0\"}"
```

Comment:
Couple of notes here.

* Jsonnet supports multi-line strings (denoted with, `|||`, [_c.f._][ml])
* [`kubecfg`][kf] exposes native extensions [parseYaml][py] and [manifestYaml][my] allows you to marshal back and forth from YAML.

[ml]: http://jsonnet.org/docs/tutorial.html#arith_cond
[kf]: https://github.com/ksonnet/kubecfg
[py]: https://github.com/ksonnet/kubecfg/blob/master/lib/kubecfg.libsonnet#L31
[my]: https://github.com/ksonnet/kubecfg/blob/master/lib/kubecfg.libsonnet#L43